workers/unity/Packages/com.improbable.gdk.mobile/Utility/LaunchArguments.cs

```diff
@@ -36,6 +38,19 @@ public static class LaunchArguments
             {
                 Debug.LogException(e);
             }
+#elif UNITY_IOS
```

Comment:
should this be `#if`

Reply:
no, at the beginning is a `#if UNITY_ANDROID`